llvm/test/Transforms/InstCombine/assume-align.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals smart
-; RUN: opt -S -passes=instcombine,simplifycfg < %s 2>&1 | FileCheck %s
+; RUN: opt -S -passes='instcombine<no-verify-fixpoint>,simplifycfg' < %s 2>&1 | FileCheck %s
```

Comment:
Please use the instcombine-no-verify-fixpoint function attribute instead, and add a comment why it's necessary.

Reply:
this is gone now